redbot/cogs/mod/mod.py

```diff
@@ -53,6 +53,7 @@ class Mod(
         "delete_delay": -1,
         "reinvite_on_unban": False,
         "current_tempbans": [],
+        "toggle_dm": False,
```

Comment:
Renaming this from `toggle_dm` to `dm_on_kickban` in config would make it more clear for people who will be looking at this code in future. Command name is okay, I just want you to change it internally from `toggle_dm` to `dm_on_kickban` - it should be a matter of find&replace in all files you modified.